submissions/ArthurGorbenko/html.css.practice/index.html

```diff
@@ -0,0 +1,175 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+<meta charset="utf-8">
+
+<title>Html/css practice</title>    
+
+<link rel="stylesheet" href="css/styles.css">
+
+</head>
+
+<body>
+    <main>      
+        <div class="container">
+                <div class="mail_button"><a href="" class="link">Gmail</a></div>
+                <div class="Images"><a href="" class="link">Images</a></div>
+                <input type="checkbox" id="trig" name="trigger">
+                <label for="trig" class="button" id="menu_btn">
+                         <svg viewBox="0 0 24 24" class="menu_btn">
+                            <path d="M 6 8 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z M 12 20 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z M 6 20 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z M 6 14 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z M 12 14 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z M 16 6 c 0 1.1 0.9 2 2 2 s 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 Z M 12 8 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z M 18 14 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z M 18 20 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z">
+                            </path>
+                         </svg>
+                    </label>
+                <div class="content">
+                <input type="checkbox" id="more">   
```

Comment:
I would move that `checkbox id=more` to the other place.

For now, it causes unexpected behavior. Using TAB focus jumps from top-level menu to more-button and only after that goes through the list of the popup menu.

I believe, the order focus using TAB should be as follows:
* top-level menu
* elements of the popup menu
* more button
https://youtu.be/5eW-JdeOhKg

Reply:
To be honest, i know about that behavior, i tried to fix. There is a thing: i need to declare `input` before `ul` that we need to hide. In case of using `display:flex;flex-direction: column-reverse;`( only with this we can get affect like autoscroll to the bottom of a list), only one place afford for `input`.But i open for tricks and advises.